### PR TITLE
fix(build): separate prod build from dev build

### PR DESCRIPTION
**Pull request by jessevanmuijden**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -60,17 +43,15 @@ const config = {
   },
   plugins: [
     new CopyPlugin({
-      patterns: [{ from: "assets", to: "assets" }],
+      patterns: [
+        { from: "node_modules/@biz-dev-ops/md-docs/assets", to: "assets" },
```
Comment (arjangeertsema) [inline webpack.config.js:47]: This needs to be added for dev build?

Comment (arjangeertsema) [reply webpack.config.js:47]: @jessevanmuijden this file loads the css files, must be removed? https://github.com/biz-dev-ops/web-components/blob/fix/build/website/src/index.ts

File: webpack.config.js
Comment (arjangeertsema) [inline webpack.config.js:1]: Plugins can be deleted, and therefore a lot of other code also?

**Review by arjangeertsema — CHANGES_REQUESTED** [review]

**Comment by arjangeertsema** [comment]
@jessevanmuijden you can test yourself:

- checking out: https://github.com/sbnnl/documentatie
- checking out: https://github.com/biz-dev-ops/md-docs-cli (dev branch)
- md-docs-cli and webcomponents should have the same parent folder.
- run npm lnk in md-docs-cli dir
- run md-docs in documentatie dir
- page should be green not purple.

File: webpack.config.js (hunk 1 above)
Comment (jessevanmuijden) [reply webpack.config.js:47]: no, webpack.config.dev.js is for the dev build. removing that index.ts file will break the dev website, but I can try to wrap the imports in an if-statement.

**Pull request merged**